### DISCUSSION TITLE
Will this map work with ReTerraForged?

### DISCUSSION THREAD

**Discussion by ffernn-dev** [jacobsjo/mc-datapack-map#47] [Q&A] [answered]
When a mod is uploaded, does the map just grab the datapack out of it, or can it actually run the mod's worldgen code? I'd like to look for seeds to work with the ReTerraForged mod, along with a couple datapacks, and I can't see any other tools that do that so far.

**Comment by jacobsjo** [comment, marked as answer]
It only reads it as if it were a datapack. Many worldgen mods do use vanilla datapack mechanics to do their worldgen - those do work with the mod. I don't think ReTerraForged is one of those mods though.

Instead, take a look at the [world preview mod](https://modrinth.com/mod/world-preview), that can actually run a mod's worldgen code.

**Reply by ffernn-dev** [reply]
Gotcha thanks :)
Also my bad, forgot GitHub had a discussions tab
